### PR TITLE
Native UIDevice Support

### PR DESCRIPTION
**Pull request by kkjdaniel**
Uses Native UIDevice Class
- Model
- Device Name
- System Name
- System Version

And updated the README

### CODE REVIEW

File: Device.js [hunk 1]
```diff
@@ -1,55 +1,31 @@
...
+    this.height = Dimensions.get("window").height;
+    this.model = DeviceUtil.model;
+    this.deviceName = DeviceUtil.name;
+    this.systemName = DeviceUtil.systemName;
+    this.systemVersion = DeviceUtil.systemVersion;
   }
 
   isIpad() {
-    return this.kind.indexOf("iPad") === 0;
+  	if (this.model.indexOf("iPad") >= 0) {
```
Comment (GertjanReynaert) [inline Device.js:18]: This should do the trick: 

``` javascript
return this.model.indexOf("iPad") >= 0;
```

File: Device.js [hunk 2]
```diff
@@ -1,55 +1,31 @@
...
-    }
-  }
-
-  _width() {
-    return Dimensions.get("window").width;
-  }
-
-  _height() {
-    return Dimensions.get("window").height;
+  	if (this.model.indexOf("iPhone") >= 0) {
```
Comment (GertjanReynaert) [inline Device.js:24]: same comment as for `isIpad`

**Comment by GertjanReynaert** [comment]
@kkjdaniel Dude this is awesome work :)

Since practically half the repo's code after this pr will be your work, I've made you collaborator of this project.

Apart from some minor remarks I posted, this pr is ready to merge :)

**Comment by kkjdaniel** [comment]
Great thanks!

**Pull request merged**